data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/ListToMapProcessorConfig.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.mutateevent;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ListToMapProcessorConfig {
+    enum FlattenedElement {
+        FIRST("first"),
+        LAST("last");
+
+        private final String name;
+
+        private static final Map<String, FlattenedElement> ACTIONS_MAP = Arrays.stream(FlattenedElement.values())
+                .collect(Collectors.toMap(
+                        value -> value.name,
+                        value -> value
+                ));
+
+        FlattenedElement(String name) {
+            this.name = name.toLowerCase();
+        }
+
+        @Override
+        public String toString() {
```

Comment:
I think you can delete this. It looks like some testing code.

Reply:
Done.